modulus/utils/capture.py

```diff
@@ -308,8 +367,10 @@ class StaticCaptureTraining(_StaticCapture):
 
     Note
     ----
-    Presently only a single instance of training static capture with AMP can be
-    used due to a grad scalar singleton.
+    Static captures must be checkpointed when training using the `state_dict()` if AMP
+    is being used with gradient scaler. By default, this requires static captures to be
+    instantiated in the same order as when they were checkpointed. The label parameter
+    can be used to avoid this.
```

Comment:
Change the last line to avoid confusion?
> The label parameter can be used to relax/circumvent this ordering requirement.

Reply:
Updated!